tests/integration_tests/data_steward/tools/top_heel_errors_test.py

```diff
@@ -66,7 +66,7 @@ def setUp(self):
         self.bucket: str = gcs_utils.get_drc_bucket()
```

Comment:
Is changing this to 
```suggestion
        self.bucket: str = self.storage_client.get_drc_bucket()
```
out of scope for this PR?

Reply:
Yes, because ultimately we don't want to use anything in the StorageClient that returns strings because of the plan above.  Return buckets, then migrate cases like these to buckets and not strings.

Reply:
👍